### PR TITLE
💬 volunteer app copy changes

### LINKED ISSUE #278: Minor copy changes
- [ ] Remove twine logo from register (unless it looks weird)
- [ ] Get rid of login fields placeholders
- [ ] Change "dashboard" to "your contribution"
- [ ] Change "your hours" to "you gave..."
- [ ] Add what time period the dashboard stats are calculated over
- [ ] Change "view volunteers" to "volunteers" on the nav element

### LINKED ISSUE #279: Move Project input above Activity input
In the new log and edit log views, "Project" input field should be above "Activity" input field.

### PR DESCRIPTION
**Pull request by eliascodes**
fixes #278 
fixes #279 

### Changes
- Move projects input above activities input
- Address changes mentioned #278

### Testing Requirements
- [ ] Volunteer app
  - Web version
	- run `ionic serve`
    - Observe login page has placeholders removed and input boxes enlarged
    - Login as `VOLUNTEER`
    - Observe dashboard has copy changed as per recommendations in #278 
    - Log out
    - Observe Twine logo has been removed from register page
    - Log in as `VOLUNTEER_ADMIN`
    - Observe nav bar has the the copy change as per recommendations in #278

### Release Requirements
- Wait for changes related to #280, #281, #282 to be merged
- Build app 
- release internal testing version (Android & Test Flight) (make available to PO)
- Release to app stores on PO approval

### Manual Deployment
- Volunteer app

### CODE REVIEW

**Comment by astroash** [comment]
@eliascodes as we are doing a volunteer app release can we do a feature branch for all the relevant changed to me merged into

File: volunteer-app/www/templates/edit-log.html [hunk 1]
```diff
@@ -41,6 +41,15 @@
             </label>
             <!-- /minutes -->
 
+            <!-- Project -->
+            <span class="input-label" ng-if="hasProjects">Project (optional)</span>
```
Comment (astroash) [inline volunteer-app/www/templates/edit-log.html]: There's an issue with how we are handling deleted Projects.

Testing Steps:
- add a project to a log
- see project appended to log display
- delete project
- project is no longer appended to log

This is not expected behaviour. Projects should be soft deleted

Comment (eliascodes) [reply volunteer-app/www/templates/edit-log.html]: What do you mean by "soft deleted"? When projects are deleted, `deleted_at` is set to the current date, and the foreign key linking a log to a project is set to null.

Comment (astroash) [reply volunteer-app/www/templates/edit-log.html]: From a UX point of view when you delete a project the only change you should see is that you cannot add this project to new logs.You should still be able to see a soft deleted project linked to old logs. It is part of historical data.

Comment (eliascodes) [reply volunteer-app/www/templates/edit-log.html]: OK sure, that can be done, but it doesn't have anything to do with the issues this PR is addressing (also I thought this whole deleted projects thing was an ongoing conversation).

**Pull request merged**